### PR TITLE
Started a new GPS module for Raincatcher

### PR DESCRIPTION
**Pull request by nialldonnellyfh**
# Motivation

This module allows Raincatcher application to publish and subscribe to location based topics.

This is useful if a Raincatcher application wants to know it's current location.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Feedhenry Raincatcher GPS module
+
+This is a simple Raincatcher module that provides a service that subscribes and publishes location topics.
+
+## Topics
+
+### Subscription
+
+This module subscribes to the `wfm:gps:position:update` topic. This triggers a call to `navigator.geolocation` functionality in the browser if available.
```
Comment (danbev) [inline README.md]: Nit: Should this be: "This triggers a call to **the** `navigator.geolocation` functionality in the browser if available?"

**Review by danbev — APPROVED** [review]
LGTM Added a very minor comment/suggestion

File: README.md (hunk 1 above)
Comment (nialldonnellyfh) [reply README.md]: Yep, thanks.

**Review by ghost — COMMENTED** [review]
LGTM

**Pull request merged**